lua/lspconfig/volarvuels.lua

```diff
@@ -0,0 +1,88 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local server_name = 'volarvuels'
+
+-- TODO: This is the name if you download volar-server from the AUR
+-- need to wait for https://github.com/johnsoncodehk/volar/issues/458
+local bin_name = 'volar-server'
+
+-- TODO: Figure out how to load this dynamically
+local typescript_server_path = '/home/artur/.npm-global/lib/node_modules/typescript/lib/tsserverlibrary.js'
+
+local volar_init_options = {
+  typescript = {
+     serverPath = typescript_server_path,
+   },
+   languageFeatures={
+     references = true,
+     definition = true,
+     typeDefinition = true,
+     callHierarchy = true,
+     hover = true,
+     rename = true,
+     renameFileRefactoring = true,
+     signatureHelp = true,
+     codeAction = true,
+     completion={
+       defaultTagNameCase='both',
+       defaultAttrNameCase='kebabCase',
```

Comment:
The lack of spaces between '=' will probably trigger a linting error. I didn't run the linters yet